Library/Homebrew/cmd/audit.rb

```diff
@@ -1,87 +1,76 @@
-#:  * `audit` [`--strict`] [`--online`] [<formulae>]:
+#:  * `audit` [`--strict`] [`--online`] [`--display-cop-names`] [<formulae>]:
 #:    Check <formulae> for Homebrew coding style violations. This should be
 #:    run before submitting a new formula.
 #:
 #:    If no <formulae> are provided, all of them are checked.
 #:
-#:    If `--strict` is passed, additional checks are run. This should be used
-#:    when creating for new formulae.
+#:    If `--strict` is passed, additional checks are run, including Rubocop
```

Comment:
Alright, switching into super-pedantic mode: The project name is “Rubo**C**op”, not “Rubocop”. (This is just one of several instances.)